README.md

```diff
@@ -88,3 +90,32 @@ This space is for the main description. Use it wisely.
 * Including tests and docs, again.
 
 ```
+
+## Defining FAQs
+
+Users can also add FAQs by simply creating a markdown file in _faq folder.
+
+Format for faq markdown file is as follows: 
+```
+---
+Question: <Write the question here>
+---
+
+Answer
+```
+
+## Multi Language Support
+
+coala Projects supports multiple languages. To add a translation of a project
```

Comment:
you may want to mention "human languages" so that the distinction is clear since we support so many programming languages too ;)